camera_info_manager/include/camera_info_manager/camera_info_manager.hpp

```diff
@@ -178,20 +185,23 @@ class CameraInfoManager
   CameraInfoManager(
     rclcpp::Node * node,
     const std::string & cname = "camera",
-    const std::string & url = "");
+    const std::string & url = "",
+    const std::string & ns = "~");
 
   CAMERA_INFO_MANAGER_PUBLIC
   CameraInfoManager(
     rclcpp_lifecycle::LifecycleNode * node,
     const std::string & cname = "camera",
-    const std::string & url = "");
+    const std::string & url = "",
+    const std::string & ns = "~");
 
   CAMERA_INFO_MANAGER_PUBLIC
   CameraInfoManager(
     rclcpp::node_interfaces::NodeBaseInterface::SharedPtr node_base_interface,
     rclcpp::node_interfaces::NodeServicesInterface::SharedPtr node_services_interface,
     rclcpp::node_interfaces::NodeLoggingInterface::SharedPtr node_logger_interface,
     const std::string & cname = "camera", const std::string & url = "",
+    const std::string & ns = "~",
     rmw_qos_profile_t custom_qos = rmw_qos_profile_default);
```

Comment:
This way we will not break other user 
```suggestion
    const std::string & cname = "camera", const std::string & url = "",
    rmw_qos_profile_t custom_qos = rmw_qos_profile_default,
    const std::string & ns = "~");
```